ocfweb/account/register.py

```diff
@@ -37,8 +42,14 @@ def request_account(request: HttpRequest) -> Union[HttpResponseRedirect, HttpRes
     status = 'new_request'
 
     existing_accounts = search.users_by_calnet_uid(calnet_uid)
+    eligible_group_signatories = groups_by_student_signat(calnet_uid)
```

Comment:
variable name makes it sound like a list of signatories, but it's really a list of groups. maybe `eligible_signatory_groups`?